doc/howto/network_ovn_setup.md

```diff
@@ -1,33 +1,123 @@
 (network-ovn-setup)=
 # How to set up OVN with LXD
 
+See the following sections for how to set up a basic OVN network, either as a standalone network or to host a small LXD cluster.
+
+## Set up a standalone OVN network
+
+Complete the following steps to create a standalone OVN network that is connected to the parent network (for example, `lxdbr0`) for outbound connectivity.
```

Comment:
"that is connected to a managed LXD parent bridge network (" - as the config options used further down rely on it being a LXD managed bridge.